kickstarts/partials/post/repos.ks.erb

```diff
@@ -4,6 +4,9 @@
 
 yum config-manager --set-enabled PowerTools
 yum config-manager --setopt=epel.exclude=*qpid-proton* --save
+<% if @build_type != "release" %>
+dnf config-manager --enable manageiq-*-nightly
```

Comment:
Realized we've been using `yum` instead of `dnf` in the kickstart files. I'll change everything else to use `dnf` in another PR.